vaadin-radio-button-flow-parent/vaadin-radio-button-flow-integration-tests/src/main/java/com/vaadin/flow/component/radiobutton/tests/RadioButtonGroupDemoPage.java

```diff
@@ -285,6 +287,26 @@ private void addDisabledItems() {
                 valueInfo);
     }
 
+    private void addDisabledGroupWithItemEnabledProvider() {
```

Comment:
This shouldn't be a demo. Please create a new view or reuse one of the other ones

Reply:
I moved the test to another class and modified that class so the component tested has an `ItemEnableProvider` object. It should not affect the other single test that it had and the class now covers the same case this one covered.